lib/ActivityStream.jsm

```diff
@@ -207,6 +207,7 @@ const PREFS_CONFIG = new Map([
       type: "local",
       localProvider: "OnboardingMessageProvider",
       enabled: true,
+      exclude: ["RETURN_TO_AMO_1"],
```

Comment:
I like this, let's document this or add some tests somewhere so this is easy to understand/use.